addons/mail/static/src/discuss/core/common/dropdown.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="discuss.DiscussDropdown">
+        <div class="o-discuss-DiscussDropdown" t-att-class="props.class">
+            <div t-ref="trigger" t-on-click="onTriggerClick">
+                <t t-slot="default"/>
+            </div>
+
+            <div t-if="isOpen"
+                 t-ref="content"
+                 t-on-click="onContentClick"
+                 class="o-discuss-DiscussDropdown-content"
+                 t-att-class="props.contentClass">
+                <t t-slot="content"/>
+            </div>
+        </div>
+    </t>
+
+</templates>
```

Comment:
Color of dropdown is wrong in dark theme:
<img width="358" alt="Screenshot 2025-06-05 at 17 02 16" src="https://github.com/user-attachments/assets/de4738d2-33e4-4073-aad9-83c80f8c0b1f" />

I also think that the dropdown should not use the theme color: it should use static (dark) bg color with white text like the call view.